chaos.c

```diff
@@ -4,33 +4,38 @@
 
 #if USE_CHAOS_STREAM_SOCKET
 
-#include <stdio.h>
```

Comment:
Seems this is a but superfluous?  I have no opinion though ..

Reply:
Yeah, that's just GNU style (modifications via GNU Cppi tool).  If you hate it, I can revert.

Reply:
I don't see that this was resolved in the discussions.  I think minor whitespace cleanup is ok, but major reformatting is not.

Reply:
That change is from GNU Cppi which normalizes indentation to GNU style - it's indenting by one space because those preprocessor directives are inside the conditional #if block.

Reply:
Sure, but we haven't adopted GNU style here.  At least it hasn't been discussed and agreed on.

Reply:
I can revert to the previous flat style with no indentation (all preprocessor directives left justified) if it's preferred.  I apologize, it's something that I (and my default editor configuration) do automatically.

Reply:
Oh no, need to apologize!  We have these reviews and discussions to hash things out.

In my view, reformatting be more nuisance than it's worth.  Say, if someone patched a fork and wanted to apply it here later.  It's also polite to adopt the existing code style.  So that's my preference, but I'm not saying I have the final say.

Reply:
OK, I'll still lean to the new formatting due to the (IMO, massive) readability improvements, but let me know either way.

Reply:
I'm with @larsbrinkhoff -- one benefit of keeping formatting is that when you compare to old versions of supdup that might surface it is much much easier to find changes.

As for GNU cppi .. I personally find it ugly, and it isn't that commonly adopted in GNU.

Reply:
Please undo the reformatting.

Reply:
Done.